README.md

```diff
@@ -1,6 +1,48 @@
 # PSR - Log Util
 Utilities for the [`psr/log`][] package that backs the [PSR-3][] specification.
 
+ * `Psr\Log\Test\LoggerInterfaceTest` provides a base test class for ensuring compliance with the LoggerInterface.
+ * `Psr\Log\Test\TestLogger` is a mock class for testing purposes.
+
+## Installation
+
+This package should be used only for tests.
+
+For PHP 8.0+, you may remove support for older versions of `psr/log`:
+
+```json
+{
+  "require": {
+    "php": ">=8.0",
+    "psr/log": "^3.0"
+  },
+  "require-dev": {
+    "psr/log-utils": "^1.1"
+  }
+}
+```
+
+If the project supports older versions of PHP, you can accept all versions:
+
+```json
+{
+  "require": {
+    "psr/log": "^1.1.14 | ^2.0 | ^3.0"
+  },
+  "require-dev": {
+    "psr/log-utils": "^1.0"
+  }
+}
+```
+
+## Versions
+
+The version of `psr/log-utils` installed varies with the version of `psr/log`.
```

Comment:
```suggestion
The version of `psr/log-utils` required depends on the version of `psr/log` that you are using:
```

Reply:
I'd like to make clear that this is the version that composer decides to install after dependency resolution. Not the versions that are set in the `require` section of `composer.json`.